### PR TITLE
Places: Discovery with tooltips

### PR DESCRIPTION
**Pull request by sharvaniharan**
**Phab:** https://phabricator.wikimedia.org/T355302

- Builds 2 tooltips: One on main nav tab and one for page more menu
- Removes Watchlist main and page tooltips and related resources completely.
- Added dev preferences for new tooltips

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1686,7 +1686,7 @@
     <string name="places_toggle_buttons_map">Map</string>
     <string name="places_toggle_buttons_list">List</string>
     <string name="places_nav_tab_tooltip_message">Discover Wikipedia articles around you with Places</string>
-    <string name="places_article_menu_tooltip_message"><![CDATA[Tap <b>\'View on map\'</b> to explore this article\'s location]]></string>
+    <string name="places_article_menu_tooltip_message"><![CDATA[Tap \'View on map\' to \nexplore this article\'s location]]></string>
```
Comment (dbrant) [inline app/src/main/res/values/strings.xml:1689]: Do we actually need an explicit newline `\n` there? Shouldn't we allow the text to overflow normally?

**Review by dbrant — CHANGES_REQUESTED** [review]
Logic looks good, just one tiniest nit.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (sharvaniharan) [reply app/src/main/res/values/strings.xml:1689]: It makes the tooltip very wide like this, so I had added line-break to get closer to the designs:
long_tooltip

**Comment by dbrant** [comment]
more code reduction = excellent

**Pull request merged**